polling_stations/apps/data_collection/management/commands/import_bedford.py

```diff
@@ -4,33 +4,42 @@
 class Command(BaseDemocracyCountsCsvImporter):
     council_id = "E06000055"
     addresses_name = (
-        "europarl.2019-05-23/Version 1/Democracy Club - Polling Districts.csv"
+        "parl.2019-12-12/Version 1/Democracy Club - Polling Districts UKPGE.csv"
     )
     stations_name = (
-        "europarl.2019-05-23/Version 1/Democracy Club - Polling Stations.csv"
+        "parl.2019-12-12/Version 1/Democracy Club - Polling Stations UKPGE.csv"
     )
-    elections = ["europarl.2019-05-23"]
+    elections = ["parl.2019-12-12"]
+    allow_station_point_from_postcode = False
 
     # KEMPSTON WEST METHODIST CHURCH Carried forward from local.2019-05-02
     def station_record_to_dict(self, record):
-        if record.stationcode == "50":
-            record = record._replace(xordinate="0")
-            record = record._replace(yordinate="0")
+        if record.stationcode == "BAS_1":
+            record = record._replace(xordinate="502614")
+            record = record._replace(yordinate="247440")
 
         return super().station_record_to_dict(record)
 
     def address_record_to_dict(self, record):
-        rec = super().address_record_to_dict(record)
         uprn = record.uprn.strip().lstrip("0")
 
+        # Lot's of junk UPRNs
+        record = record._replace(uprn="")
```

Comment:
Gah - I hadn't spotted these - I'd have bounced it back to them for a fresh export if I had.
If you're removing all the UPRNs here, how are you still matching on UPRN a file lines down with `if uprn in ["10024229957"]:`? Did you mean to just bin a subset of them?

Reply:
I thought I could get away with it by grabbing it in line 24 before I delete it. Do you want me to pull it out and send it back anyway?

Reply:
I get that you can set the "accept_suggestion" flag on the right record because you've grabbed the UPRN to a local var before binning it, but what I mean is if `rec` doesn't have a UPRN on it, there's no suggestion to accept. By the time you've finished with it, the dict is

```py
{
    'address': 'SKYLARKS, KIMBOLTON ROAD, BOLNHURST, BEDFORD',
    'postcode': 'MK44 2EY',
    'polling_station_id': 'NAV_1',
    'uprn': '',
    'accept_suggestion': True
}
```

..so you've told it to accept a suggestion, but it has no UPRN to look up in addressbase and make a suggestion with so we need to do the correction manually. Setting the flag probably does silence the warning even if we can't do anything with it though - lol.

If you reckon this one is pretty tidy apart from the UPRNs, I don't mind rolling with it. If you reckon its masking a bunch of problems or making it hard to check, we can go back and ask for a new export on Monday - I'll defer to your judgement.

Reply:
OK that makes sense. I'll have a look.